src/api.js

```diff
@@ -48,6 +48,28 @@ async function getOrgName(id) {
   return null;
 }
 
+/**
+ * Fetches the details of an organization from the steps app database
+ * @param {number} id 
+ */
+async function getOrg(id) {
+  const org = await rp({
+    method: 'GET',
+    uri: `${process.env.API_URL}/orgs/${id.toString()}`,
+    headers: {
+      Authorization: `Bearer ${process.env.OAUTH_ACCESS_TOKEN}`,
+    },
+    json: true
+  }).catch((e) => {
+    console.log('getOrg api method failed for org id ' + id, e.message);
```

Comment:
I don't think this `console.log` is necessary as we're already handling the error.

Reply:
The console.log is actually necessary. We have a Heroku add-on (paper-trial) that caches the console errors and saves them on our dashboard and also forward it to roo email, to enable us to understand what might have gone wrong after the app has been deployed.